### PR TITLE
Redirect people to the new wiki

### PR DESCRIPTION
**Pull request by AnotherPillow**
Add a check if someone links `stardewcommunitywiki.com`, and links the user to `stardewvalleywiki.com` instead.

### CODE REVIEW

File: src/main.py [hunk 1]
```diff
@@ -179,6 +179,12 @@ async def on_message(message: discord.Message):
         log_info = parse_log(log_link)
         await message.channel.send(log_info)
 
+    for community_wiki_link in re.findall(r"https://stardewcommunitywiki.com/[a-zA-Z0-9_/:-%]+", message.content):
+        link_path = urllib.parse.urlparse(community_wiki_link).path
+        new_url = "https://stardewvalleywiki.com" + link_path
```
Comment (aquova) [inline src/main.py]: I think we would want a `/` between the hardcoded URL and `link_path`. There's a `urllib.urljoin` which would be good here.

File: src/main.py [hunk 2]
```diff
@@ -179,6 +179,12 @@ async def on_message(message: discord.Message):
         log_info = parse_log(log_link)
         await message.channel.send(log_info)
 
+    for community_wiki_link in re.findall(r"https://stardewcommunitywiki.com/[a-zA-Z0-9_/:-%]+", message.content):
```
Comment (aquova) [inline src/main.py]: I think the regex could be `https://stardewcommunitywiki.com/[a-zA-Z0-9_/:-%]*` (replacing the + with a *) as we also want to catch the home page as well.

**Review by aquova — COMMENTED** [review]
A few minor suggestions about regex

**Comment by AnotherPillow** [comment]
All done

**Review by aquova — APPROVED** [review]

**Pull request merged**